pkg/client/object.go

```diff
@@ -0,0 +1,38 @@
+package client
+
+import (
+	"fmt"
+	"sort"
+
+	runtimeclient "sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+type CompareObjects func(firstObject, secondObject runtimeclient.Object) (bool, error)
```

Comment:
So, is this really used somewhere? if so, then where? 
https://github.com/codeready-toolchain/toolchain-common/pull/211#discussion_r716450286

Reply:
indeed, it's not (I forgot to check and delete it)

Reply:
removed in 0676b94